code/__DEFINES/is_helpers.dm

```diff
@@ -185,6 +185,8 @@
 
 #define isidcard(A) (istype(A, /obj/item/card/id))
 
+#define isainode(A) (istype(A, /obj/effect/ai_node))
```

Comment:
Put this under ´#define iseffect(O) (istype(O, /obj/effect))´ instead on in the middle of the item defines